save-backend/src/test/kotlin/com/saveourtool/save/backend/controller/OrganizationControllerTest.kt

```diff
@@ -48,7 +48,7 @@ import java.util.*
     ProjectPermissionEvaluator::class,
     LnkUserProjectService::class,
     UserDetailsService::class,
-
+    OriginalLoginRepository::class,
```

Comment:
Should be either as `@Import` or `@MockBean`

Reply:
Fixed